types.ml

```diff
@@ -29,7 +54,9 @@ module Change_with_hint = struct
     | `Add_test_dep _ -> true
 
   let pp f (c, dirs) =
-    let dirs = Dir_set.map (function "." -> "/" | x -> x) dirs in
+    let dirs =
+      Dir_set.map (fun path -> if Fpath.is_current_dir path then Fpath.v "/" else path) dirs
```

Comment:
Are you trying to get the root directory? This won't work on Windows, where the root is not `/` (nor `\` for that matter. If you _really_ need the root, you could get the current working directory with `Sys.getcwd ()` and extract the root from that path. If you have another way to do what you want without using the root, you should try that. If you want the directory separator you could use `Filename.dir_sep` or `Fpath.dir_sep` for clarity.

Reply:
The purpose is to print information. This is not supposed to be used.